src/dumpling-service/dumpling.web/Controllers/DumplingServiceController.cs

```diff
@@ -38,6 +41,16 @@ public class DumplingServiceController : ApiController
             return await StateTableController.GetState(new StateTableIdentifier() { Owner = owner, DumplingId = dumplingid });
         }
 
+        [Route("dumpling/property/add/{dumplingid}")]
+        [HttpPost]
+        public async Task AddProperty(int dumplingid, [FromBody]JToken jsonData)
+        {
```

Comment:
I think we should keep the naming consistent here i.e. AddTriageInfo with the route "dumpling/triage/add/{dumplingid}" or something like that.